firmware/application/apps/ui_scanner.hpp

```diff
@@ -116,13 +116,16 @@ class ScannerView : public View {
     uint32_t browse_wait{5};
     uint32_t lock_wait{2};
     int32_t squelch{-30};
+    scanner_range_t frequency_range{0, MAX_UFREQ};
     app_settings::SettingsManager settings_{
         "rx_scanner"sv,
         app_settings::Mode::RX,
         {
             {"browse_wait"sv, &browse_wait},
             {"lock_wait"sv, &lock_wait},
             {"scanner_squelch"sv, &squelch},
+            {"range_min"sv, &frequency_range.min},
+            {"range_max"sv, &frequency_range.max},
```

Comment:
I'm pretty happy with how easy that ended up being to use. :)

Reply:
Yes, it looks nice!
Thanks Kal :-)